docs/platforms.md

```diff
@@ -0,0 +1,41 @@
+# Platforms
+
+This document outlines the supported platforms for matrix.org bridge projects.
+
+### Bridge versioning
+
+All bridge application and library projects aim to follow the [Semantic Versioning](https://semver.org/) system.
+
+### Node version support
+
+Bridge projects will always support the **Active LTS** release and the **Current** release of Node.JS. When a
+Node.JS version falls from Active to Maintenance, we will migrate projects to the next Active release over time.
+
+At time of writing (July 2022) we support 16.X and 18.X as they are the Active and Current releases of Node.
+
+Bridge projects do not support odd-versioned Node.JS releases, as these are short lived non-LTS versions and are
+difficult to support since they have a 6 month shelf life.
+
+See https://nodejs.org/en/about/releases/ for more information about Node.JS releases.
+
+### LTS releases
+
+We do not currently have a LTS process for bridge projects. The latest released version of the bridge is
+what we aim to support. Versions a few points behind latest might be eligible for some support, but the
+advice is always to upgrade as soon as possible.
+
+### Platforms
+
+In terms of installation platforms, we support as a baseline:
+
+- Installation onto a host directly (via git) on Windows, Mac, Linux.
+- Docker (x86_64) on Windows, Mac, Linux.
```

Comment:
I think we should specify architecture in both cases, if applicable - unless it only matters in case of docker?

Maybe also add a note saying "we test things on Debian Stale/Ubuntu LTS/whatever, but we'll help you anyway with whatever distro you have" (which I think we can safely commit to until proven otherwise).

Reply:
Yeah, might as well say we have tested on x86_64 Debian ( & derivatives) but we expect any Linux distro to work. In addition, other architectures will *probably* be fine.

Reply:
>I think we should specify architecture in both cases, if applicable - unless it only matters in case of docker?

Ah, should clarify. We build x84_64 images for Docker automatically, but we don't specifically support an arch so others should be fine.

Reply:
Hopefully better?

Reply:
LGTM :)